### ISSUE TITLE
`invoiceitem.updated` poorly documented

### ISSUE THREAD

**Issue by paymog** [stripe/stripe-node#1984]
### Describe the bug

There seems to be an event type for `invoiceitem.updated` found [here](https://github.com/stripe/stripe-node/blob/master/types/EventTypes.d.ts#L3395-L3402) but when I view the [Event Types](https://stripe.com/docs/api/events/types) in the API reference, it's not listed. Is the SDK correct or the API Reference?

### To Reproduce

NA

### Expected behavior

NA

### OS

macos

### Node version

NA

### Library version

NA

### API version

NA

**Comment by remi-stripe** [comment]
Hey @paymog, sorry for the confusion. The Event type existed until late last year when we removed it from our API. As existing integrations still relied on this Event, we kept it for them for a while. It's now fully removed from our API for everyone but we haven't yet deprecated it from SDKs as this will require a major version release. It's slotted to be removed in the next major version we release.
In the meantime we'll update the docstring to be clearer about this! I'll tag as future for now so that we don't forget to remove it!
